presto-main/src/main/java/com/facebook/presto/cost/JoinStatsRule.java

```diff
@@ -243,10 +244,12 @@ private PlanNodeStatsEstimate filterByAuxiliaryClause(PlanNodeStatsEstimate stat
                 .setDistinctValuesCount(retainedNdv)
                 .build();
 
-        return stats
-                .mapSymbolColumnStatistics(clause.getLeft(), oldLeftStats -> newLeftStats)
-                .mapSymbolColumnStatistics(clause.getRight(), oldRightStats -> newRightStats)
-                .mapOutputRowCount(rowCount -> rowCount * UNKNOWN_FILTER_COEFFICIENT);
+        PlanNodeStatsEstimate.Builder result = PlanNodeStatsEstimate.buildFrom(stats);
+        result.setOutputRowCount(stats.getOutputRowCount() * UNKNOWN_FILTER_COEFFICIENT);
+        result.addSymbolStatistics(clause.getLeft(), newLeftStats);
+        result.addSymbolStatistics(clause.getRight(), newRightStats);
```

Comment:
continuing https://github.com/prestodb/presto/pull/11856#discussion_r232168764

you still can use builder fluently like below; this will be more readable IMO
```
PlanNodeStatsEstimate.Builder result = PlanNodeStatsEstimate.buildFrom(stats)
    .setOutputRowCount(stats.getOutputRowCount() * UNKNOWN_FILTER_COEFFICIENT)
    .addSymbolStatistics(cla ...

return normalizer.normalize(result.build(), types);
```

Reply:
Oh, that's what you meant. Agreed.